pkgs/desktops/mate/mate-tweak/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+, intltool
+, mate
+, libnotify
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+, glib
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mate-tweak";
+  version = "20.04.0";
+
+  src = fetchFromGitHub {
+    owner = "ubuntu-mate";
+    repo = pname;
+    rev = version;
+    sha256 = "14cvh5hf61s0z75skrpz6r0z27jd01z100v2cqih2ck0pxv0483v";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    intltool
+    python3Packages.distutils_extra
+    gobject-introspection
+  ];
+
+  buildInputs = [
```

Comment:
This crashes on a non mate system with
```

(mate-tweak:27457): GLib-GIO-ERROR **: 14:56:12.502: Settings schema 'org.mate.session.required-components' is not installed
```

we need to put `mate-session` in `buildInputs` to provide the schema

Reply:
`mate.mate-session-manager`?
It doesn't crash on my system, I use i3wm.